tests/test_evaluation.py

```diff
@@ -13,13 +13,16 @@ def test_create_result() -> None:
     """Test the creation of a result object."""
     devices = get_available_devices()
     assert devices[0].num_qubits > 10
-    res = create_tket_result(QuantumCircuit(10), devices[0])
+    qc = QuantumCircuit(10)
+    for i in range(10):
+        qc.id(i)
+    res = create_tket_result(qc, devices[0])
     assert isinstance(res, Result)
     assert res.compilation_time >= 0.0
     assert res.fidelity >= 0.0
     assert res.critical_depth >= 0.0
```

Comment:
Can we also include the estimated success probability in the assert statements similar to the other figures of merit?